[2025-04-10T07:59:57.029+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-10T07:59:52.934722+00:00 [queued]>
[2025-04-10T07:59:57.041+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-10T07:59:52.934722+00:00 [queued]>
[2025-04-10T07:59:57.042+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-10T07:59:57.135+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extract_lastfm> on 2025-04-10 07:59:52.934722+00:00
[2025-04-10T07:59:57.141+0000] {standard_task_runner.py:60} INFO - Started process 54 to run task
[2025-04-10T07:59:57.145+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'extract_lastfm', 'manual__2025-04-10T07:59:52.934722+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmptdnqz5pb']
[2025-04-10T07:59:57.146+0000] {standard_task_runner.py:88} INFO - Job 231: Subtask extract_lastfm
[2025-04-10T07:59:57.248+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-10T07:59:52.934722+00:00 [running]> on host 72f9073dcedb
[2025-04-10T07:59:58.169+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='extract_lastfm' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T07:59:52.934722+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-10T07:59:52.934722+00:00'
[2025-04-10T07:59:58.171+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-10T07:59:58.175+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/extract_lastfm.py']
[2025-04-10T07:59:58.198+0000] {subprocess.py:86} INFO - Output:
[2025-04-10T08:00:06.370+0000] {subprocess.py:93} INFO - ‚ùå Error al extraer Last.fm: Invalid column metadata (corrupt file?)
[2025-04-10T08:00:06.506+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-10T08:00:06.581+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=extract_lastfm, execution_date=20250410T075952, start_date=20250410T075957, end_date=20250410T080006
[2025-04-10T08:00:06.775+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-10T08:00:06.828+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
